lib/graphql/dataloader/source.rb

```diff
@@ -76,17 +90,24 @@ def pending?
       # @api private
       # @return [void]
       def run_pending_keys
+        if @fetching_keys.any?
```

Comment:
`[false, nil].any?` returns `false`.
I think It should be `unless @fetching_keys.empty?` if dataloader accepts `false` or `nil` as a key.

Reply:
Ahhh thank you... I _always_ miss this one 🤦